packages/react-components/src/organisms/__tests__/TeamGroupCard.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { createListGroupResponse } from '@asap-hub/fixtures';
+import userEvent from '@testing-library/user-event';
+
+import TeamGroupTools from '../TeamGroupCard';
+
+it('renders team group tools with 1 group', () => {
```

Comment:
would prefer not use just use get by role button, the test is easier to understand if we actually mention the show more text, and it won't run into problems if the component gets a second button that does something completely different